spec/integration/aws_internet_gateway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
```

Comment:
Will you add a spec that tests the scenario of switching an internet gateway from VPC1 to VPC2?  This should test the `update_aws_object` method